src/tooltip/Tooltip.js

```diff
@@ -315,12 +348,17 @@ Tooltip.defaultProps = {
   skipAndroidStatusBar: false,
   ModalComponent: Modal,
   closeOnlyOnBackdropPress: false,
+  withKeyboardAvoidView: true,
```

Comment:
IMO the default value should be `false` as the general use of `ToolTip` is to show info as compared to this specific use case where its been used to take the user input. 
Keeping it `false` will improve the performance by not adding those containers.

What's your view regarding this. Happy to discuss :)

Reply:
@pranshuchittora Thank you very much for the suggestion.
you are correct . "Keeping it false will improve the performance by not adding those containers." :+1: 
I have now changed the default value to "false"